care/utils/filters/number_in_filter.py

```diff
@@ -0,0 +1,5 @@
+from django_filters import BaseInFilter, NumberFilter
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'django_filters'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5344459)